app/controllers/users_controller.rb

```diff
@@ -8,15 +8,32 @@ def show; end
 
   def edit; end
 
-  def update
-    if @user.update(user_params)
-      redirect_to @user
+  def personal_info; end
+
+  def show_camps
+    @camps = Camp.active
+  end
+
+  def select_camp
+    return redirect_to user_personal_info_path, notice: "You have already selected camp" if CampUser.find_by(user_id: params[:id]).present?
+
+    @camp_user = CampUser.new(camp_users_params)
+    @camp_user.user_id = params[:id]
```

Comment:
what if i intentionally send someone else's id by manipulating front end? Would this assign camp to that user?

Reply:
yes! i should make use of current_user here instead